File: seed/rust-sdk/examples/no-custom-config/dynamic-snippets/example16.rs
use seed_examples::{ClientConfig, ExamplesClient};

#[tokio::main]
async fn main() {
    let config = ClientConfig {
        base_url: "https://api.fern.com".to_string(),
        api_key: Some("<token>".to_string()),
    };
    let client = ExamplesClient::new(config).expect("Failed to build client");
    client.service_create_movie(serde_json::json!({"id":"id","prequel":"prequel","title":"title","from":"from","rating":1.1,"type":"movie","tag":"tag","book":"book","metadata":{"metadata":{"key":"value"}},"revenue":1000000})).await;
}

File: seed/java-sdk/examples/inline-file-properties/src/main/java/com/snippets/Example16.java
package com.snippets;

import com.seed.examples.SeedExamplesClient;
import com.seed.examples.resources.types.types.Movie;
import java.util.HashMap;

public class Example16 {
    public static void main(String[] args) {
        SeedExamplesClient client = SeedExamplesClient.builder()
                .token("<token>")
                .url("https://api.fern.com")
                .build();

        client.service()
                .createMovie(Movie.builder()
                        .id("id")
                        .title("title")
                        .from("from")
                        .rating(1.1)
                        .tag("tag")
                        .revenue(1000000L)
                        .prequel("prequel")
                        .book("book")
                        .metadata(new HashMap<String, Object>() {
                            {
                                put("metadata", new HashMap<String, Object>() {
                                    {
                                        put("key", "value");
                                    }
                                });
                            }
                        })
                        .build());
    }
}
